<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileEditor</class>
 <widget class="QWidget" name="ProfileEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EDIT PROFILE</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="SegmentsGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>561</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Time Segments</string>
   </property>
   <widget class="QListWidget" name="SegmentsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSegmentBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Segment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveSegmentBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QWidget" name="SegmentDetailsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="StartTimeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Time:</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="StartTimeEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="ICRLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Insulin-to-Carb:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ICRSpinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="CFLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Correction Factor:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="CFSpinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.500000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="TargetBGLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Target Glucose:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="TargetBGSpinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>4.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>5.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="BasalRateLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Basal Rate:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="BasalRateSpinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Profile</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
